Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM php:7.1-apache
+MAINTAINER Fork CMS <info@fork-cms.com>
+
+# Enable Apache mod_rewrite
+RUN a2enmod rewrite
+
+# Install the dependencies and PHP extensions needed
+RUN apt-get update && \
```

Comment:
I would combine the update and install commands, but I would split them in separate RUNS per package.
At SumoCoders we use: https://gist.github.com/tijsverkoyen/fa096eb123f2b85ca140e25ccc2e98ff

I find it more easy to read and see which dependency there are for each package.